Mission 3/NMCA_003/NMCA_003_strings.lua

```diff
@@ -1,9 +1,455 @@
+
+            Secondary:
+                Defeat the enemy ACU.
+
+            Unlocked Tech:
+                TMD/TML - once player scouts enemy TML (on the north island)
+
+        Mission 4:
+            Primary:
+                Destroy all 4 Orbital canons.
+        
+        Outro:
+            Cinematics at the ship leaving the planet.
+--]]
+
+
+
+-------------
 -- Debriefing
+-------------
+-- Showed on the score screen if player wins
 Debriefing_Win = {
     {text = 'You win', faction = 'UEF'},
 }
 
+-- Showed on the score screen if player loses
 Debriefing_Lose = {
     {text = 'You lose', faction = 'UEF'},
 }
--- Main Story
\ No newline at end of file
+
+
+
+-------------
+-- Win / Lose
+-------------
+-- When the ship dies
+ShipDestroyed = {
+    {text = '[Nichols]: The ship has been killed, mission failed.', vid = 'N03_ShipDestroyed.sfd', bank = 'N03_VO', cue = 'ShipDestroyed', faction = 'UEF'},
+}
+
+-- Player dies
+PlayerDies = {
+    {text = '[Nichols]: Player dead.', vid = 'N03_PlayerDies.sfd', bank = 'N03_VO', cue = 'PlayerDies', faction = 'UEF'},
+}
+
+
+
+------------
+-- Mission 1
+------------
+-- Look at the Aeon base
+M1Intro1 = {
+    {text = '[Benson]: We\'re picking up the ship\'s beacon somewhere from this position.', vid = 'N03_M1Intro1.sfd', bank = 'N03_VO', cue = 'M1Intro1', faction = 'UEF'},
+}
+
+-- Move cam to the crashed ship
+M1Intro2 = {
+    {text = '[Benson]: I\'ve got something, it\'s coming from the north.', vid = 'N03_M1Intro2.sfd', bank = 'N03_VO', cue = 'M1Intro2', faction = 'UEF'},
+}
+
+-- Spawn ACU and move cam to it
+M1Intro3 = {
+    {text = '[Nichols]: There\'s the ship. We have no time to waste, sending you in commander.', vid = 'N03_M1Intro3.sfd', bank = 'N03_VO', cue = 'M1Intro3', faction = 'UEF'},
+}
+
+
+
+-- Assign primary objectives, protect ship, kill aeon base
+M1PostIntro = {
+    {text = '[Benson]: Commander, Aeon units are closing to your position, secure the area and defend the ship.', vid = 'N03_M1PostIntro_1.sfd', bank = 'N03_VO', cue = 'M1PostIntro_1', faction = 'UEF'},
+    {text = '[Benson]: Once you establish a base, send your units to the south and destroy the Aeon base.', vid = 'N03_M1PostIntro_2.sfd', bank = 'N03_VO', cue = 'M1PostIntro_2', faction = 'UEF'},
+}
+
+-- Player kills the first Aeon base
+M1AeonBaseDestroyed = {
+    {text = '[Nichols]: The Aeon base has been destroyed, good job.', vid = 'N03_M1AeonBaseDestroyed.sfd', bank = 'N03_VO', cue = 'M1AeonBaseDestroyed', faction = 'UEF'},
+}
+
+-- Reminder #1 to kill Aeon base
+M1AeonBaseReminder1 = {
+    {text = '[Benson]: The Aeon base is still a threat, destroy it.', vid = 'N03_M1AeonBaseReminder1.sfd', bank = 'N03_VO', cue = 'M1AeonBaseReminder1', faction = 'UEF'},
+}
+
+-- Reminder #2 to kill Aeon base
+M1AeonBaseReminder2 = {
+    {text = '[Benson]: The base wont kill itself commander.', vid = 'N03_M1AeonBaseReminder2.sfd', bank = 'N03_VO', cue = 'M1AeonBaseReminder2', faction = 'UEF'},
```

Comment:
Grammar correction again for when displayed to the player:
'The base won't kill itself, Commander.'